PocketShop/Views/CustomerViews/CustomerOrderScreen.swift

```diff
@@ -0,0 +1,232 @@
+            }
+        }
+        .frame(maxWidth: .infinity)
+    }
+
+    @ViewBuilder
+    func OrderItem(order: AdaptedOrder) -> some View {
+        HStack(alignment: .top) {
+            VStack {
+                Text("COLLECTION NO.")
+                    .font(.appBody)
+
+                Spacer()
+
+                Text("\(order.collectionNo)")
+                    .font(.appFont(size: 32))
+                    .bold()
+
+                Spacer()
+
+                Text("\(order.orderDateString)")
+                    .font(.appBody)
+
+                Text("\(order.orderTimeString)")
+                    .font(.appBody)
+                    .foregroundColor(.gray)
+            }
+            .frame(minWidth: 100)
+
+            VStack(alignment: .leading, spacing: 0) {
+                Text("\(order.shopName)")
+                    .font(.appBody)
+                    .bold()
+                    .padding(.bottom, 4)
+
+                ForEach(order.orderProducts, id: \.id) { orderProduct in
+                    Text("\(orderProduct.quantity) x \(orderProduct.name)")
+                        .font(.appSmallCaption)
+                }
+
+                Spacer()
+            }
+            .padding(.leading, 8)
+
+            Spacer()
+
+            VStack {
+                Text(String(format: "$%2.f", order.total))
+                    .font(.appBody)
+                    .bold()
+                    .padding(.bottom, 12)
+
+                Spacer()
+
+                RingView(color: order.ringColor, text: order.statusAsString)
+
+                Spacer()
+            }
+            .frame(minHeight: 128)
+        }
+    }
+}
+
+extension CustomerOrderScreen {
+    enum TabView: String {
+        case current = "current"
+        case history = "history"
+    }
+}
+
+// MARK: view model
+extension CustomerOrderScreen {
+    struct AdaptedOrder: Identifiable {
+        var id: String
+        var collectionNo: Int
+        var shopName: String
+        var total: Double
+        var orderProducts: [AdaptedOrderProduct]
+        var status: OrderStatus
+        var isHistory: Bool {
+            status == .collected
+        }
+        var statusAsString: String {
+            switch status {
+            case .pending:
+                return "PENDING"
+            case .accepted:
+                return "ACCEPTED"
+            case .preparing:
+                return "PREPARING"
+            case .ready:
+                return "READY"
+            case .collected:
+                return "COLLECTED"
+            }
+        }
+        var ringColor: Color {
+            switch status {
+            case .pending, .accepted, .preparing:
+                return .gray6
+            case .ready, .collected:
+                return .success
+            }
+        }
+        var orderDate: Date
+        private let dateFormatter = DateFormatter()
+
+        var orderDateString: String {
+            dateFormatter.dateFormat = "dd/MM/yyyy"
```

Comment:
Previously it was `dd/mm/yyyy` where `mm` formats minutes and `MM` is the correct one for month